resources/splunk_install.rb

```diff
@@ -146,10 +153,9 @@ def kernel_string
   action :install do
     super()
 
-    converge_if_changed :version, :build do
-      poise_archive package_path do
-        destination install_dir
-      end
+    poise_archive package_path do
+      destination install_dir
+      only_if { changed? :version, :build }
```

Comment:
Not exactly. I changed this for the same reason, being that resources inside converge_if_changed tend to cause race conditions. If I remember correctly, in this case the :delete notification on the above remote_file would fire without waiting for the archive extraction to finish.